python/cudf/utils/utils.py

```diff
@@ -229,19 +229,25 @@ def get_dummies(df, prefix='', prefix_sep='_', cats={}, columns=None,
         to the empty string
     prefix_sep : str, optional
         separator to use when appending prefixes
+    dummy_na : bool, optional
+        Right now this is NON-FUNCTIONAL argument in rapids.
     cats : dict, optional
         dictionary mapping column names to sequences of integers representing
         that column's category. See `cudf.DataFrame.one_hot_encoding` for more
         information. if not supplied, it will be computed
+    sparse : bool, optional
+        Right now this is NON-FUNCTIONAL argument in rapids.
+    drop_first : bool, optional
+        Right now this is NON-FUNCTIONAL argument in rapids.
     columns : sequence of str, optional
         Names of columns to encode. If not provided, will attempt to encode all
         columns. Note this is different from pandas default behavior, which
         encodes all columns with dtype object or categorical
     dtype : str, optional
         output dtype, default 'float64'
     """
+    assert (dummy_na is False) and (sparse is False) and (drop_first is False)
```

Comment:
Instead of this assert, can we have a separate condition for each and raise a `NotImplementedError` if they're non-default? It will be much more obvious to the end user 😄

Reply:
@kkraus14, Changed to a custom message for each of those 3 if they are non-defaults.